src/openforms/forms/api/viewsets.py

```diff
@@ -294,7 +293,94 @@ def export(self, request, *args, **kwargs):
     def perform_destroy(self, instance):
         instance._is_deleted = True
         instance.save()
-        reversion.set_comment(_("Form deleted via the API."))
+
+
+@extend_schema(
+    parameters=[
+        OpenApiParameter(
+            name="form_uuid_or_slug",
+            location=OpenApiParameter.PATH,
+            type=str,
+            description=_("Either a UUID4 or a slug identifying the form."),
+        )
+    ]
+)
+@extend_schema_view(
+    create=extend_schema(
+        summary=_("Save form version"),
+        tags=["forms"],
+        responses={status.HTTP_201_CREATED: FormVersionSerializer},
+    ),
+    restore=extend_schema(
+        summary=_("Restore form version"),
+        tags=["forms"],
+        parameters=[
+            OpenApiParameter(
+                name="form_uuid_or_slug",
+                location=OpenApiParameter.PATH,
+                type=str,
+                description=_("Either a UUID4 or a slug identifying the form."),
+            ),
+            OpenApiParameter(
+                name="uuid",
+                location=OpenApiParameter.PATH,
+                type=str,
+                description=_("The uuid of the form version"),
+            ),
+        ],
+        responses={status.HTTP_201_CREATED: ""},
+    ),
+    list=extend_schema(
+        summary=_("List form versions"),
+        tags=["forms"],
+        parameters=[
+            OpenApiParameter(
+                name="form_uuid_or_slug",
+                location=OpenApiParameter.PATH,
+                type=str,
+                description=_("Either a UUID4 or a slug identifying the form."),
+            )
+        ],
+        request=FormVersionSerializer,
+        responses={status.HTTP_200_OK: FormVersionSerializer(many=True)},
```

Comment:
these are the defaults and should not be specified explicitly, right?

Reply:
Yes!